http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/Boot.scala

```diff
@@ -213,18 +208,15 @@ object Boot extends IOApp {
         appDependencies.google2StorageDao,
         appDependencies.publisherQueue
       )
-      val messageProcessorStream = if (leoExecutionModeConfig.backLeo) {
-        val pubsubSubscriber: LeoPubsubMessageSubscriber[IO] =
-          new LeoPubsubMessageSubscriber(appDependencies.subscriber)
-        Stream.eval(logger.info(s"starting subscriber ${subscriberConfig.projectTopicName} in boot")) ++ pubsubSubscriber.process
-          .handleErrorWith(
-            error => Stream.eval(logger.error(error)("Failed to initialize message processor in Boot. "))
-          )
-      } else Stream.eval(logger.info(s"Not starting subscriber in boot"))
 
-      val startSubscriber = if (leoExecutionModeConfig.backLeo) {
-        Stream.eval(appDependencies.subscriber.start)
-      } else Stream.eval(IO.unit)
+      // only needed for backleo
```

Comment:
It looks like this is outside of the `if (leoExecutionModeConfig.backLeo) {}` block?

Reply:
yea...initializing the object is okay...I can see if moving it inside the if is okay...it was a bit messier previously